docs/3. Flows/3. Flow Features/05. Integrate Google sheets in Flows.md

```diff
@@ -75,3 +75,50 @@ Click on `OK` and use the column name like below and access the value of cell va
 `@results.sheet.message_1`
 
 ![image](https://user-images.githubusercontent.com/32592458/219550825-82831f46-a239-4600-b95e-bf0533f0565b.png)
+
+
+## Write to google sheet
+
+Please find the flow created for testing purpose. The below flow is an example of the response from the user will be added directly to the google sheet. 
+
+![Screenshot1](https://github.com/glific/docs/assets/130072273/52aea06d-0693-4859-865b-ad0bdd1449a8)
+
+Below are the steps to write Google sheets in the Flows.
+
+1) Once the Flow start node is created. Then, drag the Red dot & create a new node. 
+
+![Screenshot2](https://github.com/glific/docs/assets/130072273/1bf8a56d-6c09-41f8-9f46-747d93309aef)
+
+2) Mention the **Sheet_Input** in the dropdown of **Result Name**.
```

Comment:
@AmitRChavan15 it it not necessary to add `sheet_input` as the result name it can be anything.